scripts/bash/gitlab_install.sh

```diff
@@ -30,7 +30,7 @@ INSTANCE_EXTERNAL_DOMAIN=`curl -H "Metadata-Flavor: Google" http://169.254.169.2
 echo "INFO: domain name is $INSTANCE_EXTERNAL_DOMAIN"
 EXTERNAL_IP=`curl -H "Metadata-Flavor: Google" http://metadata/computeMetadata/v1/instance/network-interfaces/0/access-configs/0/external-ip`
 echo "INFO: external IP is $EXTERNAL_IP"
-GITLAB_INSTALL_VERSION="14.9.3-ee"
+GITLAB_INSTALL_VERSION="15.2.1-ee"
```

Comment:
Version 15.3 is now released,
I think this variable should be  pushed on instance metadata and used from there, so it will be easy to configure.

Reply:
Added to Monday, holding PR.

Reply:
Monday Task set to review